CIScripts/Test/TestRunner.ps1

```diff
@@ -103,3 +103,36 @@ function Run-Tests {
         }
     })
 }
+
+function Run-Tests {
+    Param ([Parameter(Mandatory = $true)] [PSSessionT[]] $Sessions)
+
+    try {
+        Run-TestsWithoutLogs -Sessions $Sessions
+    }
+    catch {
+        Write-Host $_
```

Comment:
What is written here ?

Reply:
The exception which was catched (it's also printed after being rethrown, but I wanted to also print it hear, so it's close to messages printed by tests)

Reply:
ok